💄(scss) refactor scss (#1540)
openfun/richie: CHANGELOG.md

### Changed

> - Refactor scss to be more easily customizable. All scss imports are declared
>   only on the `_main.scss` file. Add the new `$r-theme-overwrite` and
>   `$palette-overwrite` scss map variables that overwrite only the differences.

Reviewer: ```suggestion
- Refactor scss color variables to be more easily customizable.
```

Author: changed

> - Refactor scss to be more easilly customizable. All scss imports are declared
>   only on the `_main.scss` file. Add the new `$r-theme-overwrite` and
>   `$palette-overwrite` scss map variables that overwrite only the differences.

Reviewer: ```suggestion
- Refactor scss to be more easily customizable. All scss imports are declared
  only on the `_main.scss` file. Add the new `$r-theme-overwrite` and
  `$palette-overwrite` scss map variables that overwrite only the differences.
```

Author: done
